R/ebp.R

```diff
@@ -215,7 +244,9 @@ ebp <- function(fixed,
                 cpus = 1,
                 custom_indicator = NULL,
                 na.rm = FALSE,
-                weights = NULL) {
+                aggregate_to = NULL,
```

Comment:
Careful with change of order of arguments. Why should the order be changed? If someone uses the function at the moment not by arguments but by order, then an error could occur since aggregate_to is now at the position of weights